tests/WebdriverClassicConfig.php

```diff
@@ -53,9 +52,16 @@ public function skipMessage($testCase, $test): ?string
             case $testCase === StatusCodeTest::class:
                 return 'Checking status code is not supported.';
 
-            case $testCase === EventsTest::class && $test === 'testKeyboardEvents' && $this->isOldChrome():
+            case [$testCase, $test] === [EventsTest::class, 'testKeyboardEvents'] && $this->isOldChrome():
                 return 'Old Chrome does not allow triggering events.';
 
+            case [$testCase, $test] === [EventsTest::class, 'testBlur']:
+            case [$testCase, $test] === [EventsTest::class, 'testFocus']:
+                return 'Focus/blur are not supported anymore.';
```

Comment:
I'm wondering whether this could be supported by executing `element.focus()` (or `element.blur()`) in JS.

Reply:
I tried it locally and they seem to work, however the [testBlur](https://github.com/minkphp/driver-testsuite/blob/f9d36b63480ce013d0eecc2fc0ce962ca956dff2/tests/Js/EventsTest.php#L66) test fails.

It kinda makes sense too, because you cannot trigger blur on a non-focused element.
If I add `$focusBlurDetector->focus();` before `$focusBlurDetector->blur();` then the test passes.

Reply:
I merged the PR in the driver testsuite